pkg/ring/model.go

```diff
@@ -182,12 +182,20 @@ func (d *Desc) TokensFor(id string) (tokens, other []uint32) {
 
 // IsHealthy checks whether the ingester appears to be alive and heartbeating
 func (i *IngesterDesc) IsHealthy(op Operation, heartbeatTimeout time.Duration) bool {
-	if op == Write && i.State != ACTIVE {
-		return false
-	} else if op == Read && i.State == JOINING {
-		return false
+	healthy := false
+
+	switch op {
+	case Write:
+		healthy = i.State == ACTIVE
```

Comment:
Can we have brackets here `(i.State == ACTIVE)` and more important for the next condition.

We had a big outage sometime ago with a `op` b `op` c and since then I want to use brackets if we're doing things like that :)